README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@atarzwell I completely agree with you - it is usually irrelevant to software development per-se, however, this is not limited to Github, nor to communication directly related to development.

@CoWinkKeyDinkInc honestly, I am not sure how to even address this in a manner that could explain it reasonably well to an average reader. 

We could formulate this to something similar to "We respect and support freedom of speech. However, there is time and place for everything - before executing your right, think, is the pull request review really the place to debate political issues? Cake recipes? Favorite cartoon characters?"

This needs refining, however. Freedom of speech is a topic so broad and so debated, it is really hard to define it's relevance and impact to this in one or two sentences. I could literally write at least 5 doctoral thesis on this topic alone...